app/controllers/users_controller.rb

```diff
@@ -0,0 +1,22 @@
+# class UsersController < ApplicationController
+#   # layout 'boilerplate'
+#   def index
+#     @users = User.order(id: :asc)
+#   end
+
+#   def show
+#     @users = User.find(params[:id])
+#   end
+# end
+
```

Comment:
- [ ] It's considered good practice to have a readable code that is clean from any commented(unused) code. Please remove this commented code.